syz-fuzzer/fuzzer.go

```diff
@@ -400,6 +400,17 @@ func (fuzzer *Fuzzer) checkNewSignal(p *prog.Prog, info *ipc.ProgInfo) (calls []
 		fuzzer.signalMu.Unlock()
 		fuzzer.signalMu.RLock()
 	}
+	diff := fuzzer.maxSignal.DiffRaw(info.Extra.Signal, 0)
```

Comment:
The fact that we put this into info.Extra causes a bunch of duplication in fuzzer. Please try to maybe put this at the end of info.Calls. Presumably it will remove lots of diffs in fuzzer.

Reply:
Tried that approach. It does remove most of the diffs, but adds a few others. However there's an issue: this appears to be very hard to debug. I think it's better clean up the diffs in this change so the code is less painful to read.

Reply:
Done.